app/views/dashboard/apps/appId/layout/appIdLayoutView.tsx

```diff
@@ -68,6 +70,12 @@ export default function AppIdLayoutView({
   const { flags } = useFeatureFlags()
   const [showSuccessModal, setShowSuccessModel] = useState<boolean>(false)
   const [showErrorModal, setShowErrorModel] = useState<boolean>(false)
+
+  const createdAtMinsDiffToday = useMemo(
```

Comment:
if we memoize this, will we get correct values that change over time?

meaning after 5 minutes will this be memoized to always show 1 minute, or will we get the correct value of 5+ minutes?

Reply:
You're right bro, my bad, I lost the part of the now date, we shouldn't use a memo for this.